site/src/components/TemplateVersionEditor/PublishTemplateVersionDialog.tsx

```diff
@@ -70,6 +72,16 @@ export const PublishTemplateVersionDialog: FC<
               disabled={isPublishing}
             />
 
+            <TextField
+              {...getFieldHelpers("message")}
+              label="Message"
+              placeholder="Write a short message about the changes you made..."
+              autoFocus
+              disabled={isPublishing}
+              multiline
+              rows={5}
```

Comment:
If the template version already has a message, will it be pre-populated here?

Reply:
Since the "pre-version" is generated by the system I'm almost sure it has an empty message 🤔 or am I wrong?